php/templates/deployment.yaml

```diff
@@ -120,12 +120,13 @@ spec:
               command:
 {{ tpl (toYaml .) $root | indent 14 }}
 {{- end }}
+{{- with .Values.nginx.lifecycle.preStop }}
           preStop:
             exec:
-              command: ["sh", "-c", "sleep {{ .Values.fpm.processControlTimeout | default 0 }}; nginx -s quit; sleep 5"]
-        {{- with .Values.fpm.processControlTimeout }}
```

Comment:
@cw-niinuma 
I thought that you didn't need to remove `{{- with .Values.fpm.processControlTimeout }}` and `{{- end }}`. Because `terminationGracePeriodSeconds` was calculated as `.Values.fpm.processControlTimeout + 10` if `.Values.fpm.processControlTimeout` was presented.

Reply:
As a result of the discussion, you no longer need to change it.